src/templates/prepared-map/usecase-detail/example/example-tree/parse.ts

```diff
@@ -0,0 +1,92 @@
+import type {
+  ComlinkListLiteralNode,
+  ComlinkLiteralNode,
+  ComlinkObjectLiteralNode,
+  ComlinkPrimitiveLiteralNode,
+} from '@superfaceai/ast';
+
+import type {
+  ExampleArray,
+  ExampleObject,
+  ExampleScalar,
+  UseCaseExample,
+} from '../usecase-example';
+
+export function parseObjectLiteral(
+  node: ComlinkObjectLiteralNode
+): ExampleObject {
+  const properties: ({ name: string } & (
+    | ExampleArray
+    | ExampleScalar
+    | ExampleObject
+  ))[] = [];
```

Comment:
I think this is clearer in what's happening here

```suggestion
  const properties: ExampleObject['properties'] = [];
```